src/traces/pie/index.js

```diff
@@ -24,7 +24,7 @@ module.exports = {
     moduleType: 'trace',
     name: 'pie',
     basePlotModule: require('./base_plot'),
-    categories: ['pie', 'showLegend'],
+    categories: ['pie-like', 'pie', 'showLegend', 'hiddenLabels'],
```

Comment:
Hmm. I don't think you're using `hiddenlabels` as a category anywhere.

Reply:
Yes. We have not use them yet.

Reply:
let's :hocho: it then

Reply:
Done in https://github.com/plotly/plotly.js/pull/3876/commits/abc3b39d159e5d4f53dc170b1d11f4fc3bc5f535.